pcdsdevices/mv_interface.py

```diff
@@ -0,0 +1,117 @@
+from bluesky.utils import ProgressBar
+from ophyd.status import wait as status_wait
+
+
+class MvInterface:
+    """
+    Define common shortcuts that the beamline scientists like for moving things
+    on the command line. There is no need for these in a scripting
+    environnment, but this is a safe space for implementing move features that
+    would otherwise be disruptive to running scans and writing higher-level
+    applications.
+    """
+    def mv(self, position, timeout=None, wait=False):
+        """
+        Absolute move to a position.
+
+        Parameters
+        ----------
+        position
+            Desired end position
+
+        timeout: number, optional
+            If provided, the mover will throw an error if motion takes longer
+            than timeout to complete. If omitted, the mover's default timeout
+            will be use.
+
+        wait: bool, optional
+            If True, wait for motion completion before returning. Defaults to
+            False.
+        """
+        self.move(position, timeout=timeout, wait=wait)
```

Comment:
I was under the impression that this interface would define `.move` as well, and all it would require was a `set` definition?

Reply:
I think the problem is we don't know how people are implementing their `move` and `set` methods. Some classes put all the smarts in `move` and some put them all in `set`. I can provide a `move` command too, and since this is inherited last it might be ok, but that runs the risk of getting hit with a `super().move` call from an arbitrary parent class.

Reply:
We can also do something ridiculous like provide both `move` and `set`, and then whichever one you omit gets provided.

Reply:
Voting to table this discussion for now. This establishes the API and will work. Refinements can be made later